Added new source of license text (#1011)
rubysec/ruby-advisory-db: LICENSE.txt

Source Vulnerability Database (http://osvdb.org). If you use ruby-advisory-db
data to build a product or a service, it is your responsibility to familiarize
yourself with the terms of their license: http://www.osvdb.org/osvdb_license

> (you may need to use a web archive service, such as [Here](https://scancode-licensedb.aboutcode.org/osvdb.html), to view the license).

Reviewer: ```suggestion
(an archived version of this license is available at:
https://scancode-licensedb.aboutcode.org/osvdb.html)
```
